### DISCUSSION TITLE
K8s 1.28 - Formalize docs release strategy with docs localization owners

### DISCUSSION THREAD

**Discussion by Rishit-dagli** [kubernetes/sig-release#2248] [General]
Hello 👋, I'm Rishit Dagli, the SIG Release Docs lead for the 1.28 release. I'm opening this discussion to coordinate the docs release (scheduled tentnatively for [August 15, 2023](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.28#timeline) with localization owners. I look forward to working with everyone! Let me know if you have any questions! Thank you! 🙏

cc: @kubernetes/sig-docs-fr-owners @kubernetes/sig-docs-de-owners @kubernetes/sig-docs-hi-owners @kubernetes/sig-docs-id-owners @kubernetes/sig-docs-ja-owners @kubernetes/sig-docs-ru-owners @kubernetes/sig-docs-pl-owners @kubernetes/sig-docs-ko-owners @kubernetes/sig-docs-pt-owners @kubernetes/sig-docs-es-owners @kubernetes/sig-docs-zh-owners @kubernetes/sig-docs-vi-owners @kubernetes/sig-docs-uk-owners @kubernetes/sig-docs-it-owners

**Comment by windsonsea** [comment]
Am I correct in understanding that all docs will be consolidated into [branch: dev-1.28](https://github.com/kubernetes/website/tree/dev-1.28)?

**Comment by Rishit-dagli** [comment]
Hello localization team leads! I don't think any action is required from you, but I wanted to let you know that we are on track for the release on August 15, 2023 and all Kubernetes website branches are up to date (`main`, `dev-1.28`). Let me know if I can help with anything! Thanks!
